services/chat/api.go

```diff
@@ -192,32 +208,138 @@ func (api *API) GetChats(ctx context.Context) (map[string]ChannelGroup, error) {
 			BanList:                 community.Description().BanList,
 			Encrypted:               community.Encrypted(),
 			CommunityTokensMetadata: community.Description().CommunityTokensMetadata,
+			UnviewedMessagesCount:   totalUnviewedMessageCount,
+			UnviewedMentionsCount:   totalUnviewedMentionsCount,
 		}
 
 		for t, i := range community.Images() {
 			chGrp.Images[t] = images.IdentityImage{Name: t, Payload: i.Payload}
 		}
 
-		for _, cat := range community.Categories() {
-			chGrp.Categories[cat.CategoryId] = communities.CommunityCategory{
-				ID:       cat.CategoryId,
-				Name:     cat.Name,
-				Position: int(cat.Position),
-			}
+		result[community.IDString()] = chGrp
+	}
+
+	return result, nil
+}
+
+func (api *API) GetChatsByChannelGroupID(ctx context.Context, channelGroupID string) (*ChannelGroup, error) {
```

Comment:
I noticed that the function can return chats with an empty members list, which goes against the 'rule of least surprise' and may be confusing for clients who rely on this property without anticipating such cases. To address this, we could consider introducing a new type, such as 'ChatInfo', that contains only essential fields. It may be a good idea to implement it in the future to ensure a more explicit and consistent representation of the data.